src/main/java/io/fabric8/jenkins/openshiftsync/GlobalPluginConfiguration.java

```diff
@@ -185,6 +191,12 @@ protected void doRun() throws Exception {
           buildConfigWatcher.start();
           buildWatcher = new BuildWatcher(namespaces);
           buildWatcher.start();
+          configMapWatcher = new ConfigMapWatcher(namespaces);
+          configMapWatcher.start();
+          imageStreamWatcher = new ImageStreamWatcher(namespaces);
+          imageStreamWatcher.start();
+          //imageStreamTagWatcher = new ImageStreamTagWatcher(namespaces);
+          //imageStreamTagWatcher.start();
```

Comment:
I'm working on it right now, we definitely need this feature for the card.

Something is printing logs every now and again with it, which can flood the logger after a while. It works, though, so we can uncomment it :)